scalability_and_performance/dummy-assembly.adoc

```diff
@@ -0,0 +1,23 @@
+[id="cnf-create-performance-profiles]
+= Creating a performance profile
+include::_attributes/common-attributes.adoc[]
+:context: cnf-create-performance-profiles
+toc::[]
```

Comment:
```suggestion

toc::[]
```

Reply:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata



You must have a blank line here before the toc.